Update brazil.rst (#13630)
odoo/documentation: content/applications/finance/fiscal_localizations/brazil.rst

E-Invoice for goods (NF-e)
~~~~~~~~~~~~~~~~~~~~~~~~~~
.. important::
   The Avalara integration works on a credit-based system, where each interaction with Avalara consumes 1 credit. Below are the main credit-consuming operations:

   :guilabel:`Sales Application`:
    - Tax calculation on quotations and sales orders.

   :guilabel:`Accounting Application`:
   - Tax calculation on invoices.
   - Electronic invoice submission (NF-e or NFS-e).

   :guilabel:`Note:` If you calculate taxes in Sales and later issue the invoice in Accounting, the calculation happens twice, consuming 2 credits.

   :guilabel:`Example Workflow:`
   Sales order confirmed → 1 credit (tax calculation)

>    Invoice created and confirmed → 1 credit (tax calculation)
>    Invoice submitted → 1 credit (tax calculation) + 1 credit (submit invoice)
>    → Total: 4 credits

Reviewer: ```suggestion
   Invoice created → 1 credit (tax calculation)
   Invoice confirmed and submitted → 1 credit (tax calculation) + 1 credit (submit invoice)
   → Total: 4 credits
```

   :guilabel:`Occasional Operations: (Each step is billed separately)`

>    - Correction letter (Carta de Correção)
>    - Invoice cancellation
>    - Other tax validations

Reviewer: ```suggestion
   - :ref:`Correction letter (Carta de Correção) <localizations/brazil/correction-letter>`
   - :ref:`Invoice cancellation <localizations/brazil/invoice-cancellation>`
   - :ref:`Sales Refund via Credit Note <localizations/brazil/credit-notes>`
   - :ref:`Sales Complementary Invoice via Debit Note <localizations/brazil/debit-notes>`
   - :ref:`Invalidate invoice number range <localizations/brazil/invalidate-invoice-number-range>`
   - Other tax validations.
```
Please do the same for the NFS-e section.
Feel free to modify of add cases, but if we are gonna add the detailed consumption, it is better to add all scenarios where a credit is consumed, which is every time a API call is done, like in the added scenarios. I've added the links to the specific section within the User Doc as we have it, and it is good to refer the user to that section if needed.

For Credit Notes (sales refund) and Debit Notes (Sales Complementary), there's also a Tax Computation required, so maybe you would like to add that a Tax Computation will be consumed on those scenarios as well.
:ref:`create an account in AvaTax <localizations/brazil/avatax-account>`

E-Invoice for goods (NF-e)
~~~~~~~~~~~~~~~~~~~~~~~~~~

.. important::
   The :ref:`Avalara integration <localizations/brazil/avatax-account>` works on a credit-based

>    system, where each interaction with Avalara consumes 1 credit. Below are the main

Reviewer 2: ```suggestion
   system, where each interaction with Avalara consumes one credit. Below are the main
```
We usually write numbers from zero to nine as words.

   credit-consuming operations:

   **Sales application**

   - Tax calculation on quotations and sales orders.

   **Accounting application**

   - Tax calculation on invoices.
   - Electronic invoice submission (NF-e or NFS-e).

   **Occasional operations**: (each step is billed separately)

   - :ref:`Correction letter (Carta de Correção) <localizations/brazil/correction-letter>`
   - :ref:`Invoice cancellation <localizations/brazil/invoice-cancellation>`
   - :ref:`Sales refund via credit note <localizations/brazil/credit-notes>`
   - :ref:`Sales complementary invoice via debit note <localizations/brazil/debit-notes>`
   - :ref:`Invalidate invoice number range <localizations/brazil/invalidate-invoice-number>`
   - Other tax validations.

.. note::

>    If taxes are calculated in **Sales** and later issue the invoice in **Accounting**, the

Reviewer 2: ```suggestion
   If taxes are calculated in the **Sales** app, and the invoice is later issued in the **Accounting**, the
```
The sentence lacked a clear subject, so I tried rephrasing.

>    calculation happens twice, consuming 2 credits.

Reviewer 2: ```suggestion
   calculation happens twice, consuming two credits.
```

   **Occasional operations**: (each step is billed separately)

   - :ref:`Correction letter (Carta de Correção) <localizations/brazil/correction-letter>`
   - :ref:`Invoice cancellation <localizations/brazil/invoice-cancellation>`
   - :ref:`Sales refund via credit note <localizations/brazil/credit-notes>`
   - :ref:`Sales complementary invoice via debit note <localizations/brazil/debit-notes>`
   - :ref:`Invalidate invoice number range <localizations/brazil/invalidate-invoice-number>`
   - Other tax validations.

.. note::
   If taxes are calculated in **Sales** and later issue the invoice in **Accounting**, the
   calculation happens twice, consuming 2 credits.

.. example::

   .. graphviz::

      digraph avalara_credit_consumption {
         rankdir=TB;
         node [shape=box];

         "Sales order confirmed" -> "Invoice created" [label=" 1 credit (tax calculation)"];
         "Invoice created" -> "Invoice confirmed and submitted"
         [label=" 1 credit (tax calculation)"];
         "Invoice confirmed and submitted" -> "Credits summary"
         [label=" 1 credit (tax calculation)\n+\n 1 credit (submit invoice)"];
         "Credits summary"

>          [label="Total: 4 credits", shape=ellipse, style=filled, fillcolor=lightgrey];

Reviewer 3: I just have an error message for this, but I guess it's because it's not installed on my laptop ;)
![image](https://github.com/user-attachments/assets/0cec1d11-2b74-4b8e-8b52-2f0aaab2dae6)

E-Invoice for services (NFS-e)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
.. important::
   The Avalara integration operates on a credit-based system, where each interaction with Avalara consumes 1 credit. Below are the main credit-consuming operations:

>    :guilabel:`Sales Application`:

Reviewer: Suggested changes on the NF-e section also applies here.

E-Invoice for services (NFS-e)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. important::
   The :ref:`Avalara integration <localizations/brazil/avatax-account>` works on a credit-based
   system, where each interaction with Avalara consumes 1 credit. Below are the main
   credit-consuming operations:

   **Sales application**

   - Tax calculation on quotations and sales orders.

   **Accounting application**

   - Tax calculation on invoices.
   - Electronic invoice submission (NF-e or NFS-e).
   - Invoice status check (1 credit is consumed each time the invoice status is checked).

   **Occasional operations**: (each step is billed separately)

   - :ref:`Correction letter (Carta de Correção) <localizations/brazil/correction-letter>`
   - :ref:`Invoice cancellation <localizations/brazil/invoice-cancellation>`
   - :ref:`Sales refund via credit note <localizations/brazil/credit-notes>`
   - :ref:`Sales complementary invoice via debit note <localizations/brazil/debit-notes>`
   - :ref:`Invalidate invoice number range <localizations/brazil/invalidate-invoice-number>`
   - Other tax validations.

.. note::

>    If taxes are calculated in **Sales** and later issue the invoice in **Accounting**, the
>    calculation happens twice, consuming 2 credits.

Reviewer 2: ```suggestion
   If taxes are calculated in the **Sales** app, and the invoice is later issued in the **Accounting** app, the
   calculation happens twice, consuming two credits.
```

Its legal validity is guaranteed by the digital signature and by each Brazilian state's SEFAZ
(Secretaria da Fazenda).

.. important::
   The Avalara integration operates on a credit-based system. Each operation that involves communication with Avalara consumes 1 credit. The following operations within the Point of Sale (POS) application are subject to credit consumption:
   - :guilabel:`Tax calculation at the time of sale`.

>    - :guilabel:`Electronic invoice issuance (NF-e)`.

Reviewer: ```suggestion
   - :guilabel:`Electronic invoice issuance (NFC-e)`.
```

Its legal validity is guaranteed by the digital signature and by each Brazilian state's SEFAZ
(Secretaria da Fazenda).

.. important::
   The :ref:`Avalara integration <localizations/brazil/avatax-account>` operates on a credit-based

>    system. Each operation that involves communication with Avalara consumes 1 credit. The following

Reviewer 2: ```suggestion
   system. Each operation that involves communication with Avalara consumes one credit. The following
```

   operations within the **Point of Sale** (POS) application are subject to credit consumption:

   - Tax calculation at the time of sale
   - Electronic invoice issuance (NFC-e)

.. note::

>    Each step is billed separately. For example, calculating taxes and issuing an invoice for the
>    same POS transaction will consume 2 credits.

Reviewer 2: ```suggestion
   Each step is billed separately. For example, calculating taxes and issuing an invoice for the
   same POS transaction consume two credits.
```
